railties/test/generators/named_base_test.rb

```diff
@@ -6,6 +6,8 @@
 module ActiveRecord
   class Base
     class << self
+      undef pluralize_table_names
+      undef pluralize_table_names=
       attr_accessor :pluralize_table_names
     end
```

Comment:
One thing that worries me is the comment on line 5: "# Mock out what we need from AR::Base."
This file didn't expect the accessor to be defined (by a file being required).

It could just be that the files at the top now require some extra files, they didn't before. It's probably okay to remove it.